scripts/babel/preset-v9.js

```diff
@@ -24,7 +24,7 @@ function createModuleResolverAliases(options) {
     const tsSourceRoot = aliasTuple[0];
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27406&commitId=bbe6d960ccfed2d8aeebe5c526487751ff58c005&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example Checked.chromium.png|25| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>